### PR TITLE
warn if files are missing

### PR DESCRIPTION
**Pull request by montague**
Addresses #99

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -1,15 +1,15 @@
 %{"aws_auth": {:hex, :aws_auth, "0.2.3"},
-  "bcrypt": {:git, "https://github.com/opscode/erlang-bcrypt.git", "7515e80a5e16f62aef149709f6cc09995dd480be", []},
+  "bcrypt": {:git, "git://github.com/opscode/erlang-bcrypt.git", "7515e80a5e16f62aef149709f6cc09995dd480be", []},
```
Comment (montague) [inline mix.lock]: i'm not sure why this change happened. i checked it in to make sure it came up in PR review, instead of just resetting the mix.lock file to reflect what was in master.

@ericmj any idea why running `mix deps.get` would do this?

**Comment by whatyouhide** [comment]
@montague hey, I gave this a quick shot last week and the solution you adopted falls short when you have wildcards specified in the `:files` option (there are a bunch in the default files, like `readme*`) as `File.exists?/1` will return false for them. @ericmj suggested to warn for wildcards that match no files. What do you think?

**Comment by montague** [comment]
sounds good. thanks for taking a look. i will add that.

File: mix.lock (hunk 1 above)
Comment (ericmj) [inline mix.lock]: You are using an elixir version that creates git:// urls for github dependencies, later versions use https://. You can just skip commiting mix.lock.

**Comment by montague** [comment]
We're now warning when a file or wildcard is missing.

I debated adding a separate message to distinguish between a missing file and a missing wildcard, but in the end I opted to keep it simple.

**Comment by ericmj** [comment]
The missing files cannot be added to the metadata because that will be included in the package metadata.

**Comment by montague** [comment]
i wondered about that. will refactor.

**Comment by montague** [comment]
refactor complete.

**Comment by ericmj** [comment]
Looks good, thank you!
